src/app/components/authentication/authentication.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from 'src/app/services/auth.service';
+
+@Component({
+  selector: 'app-authentication',
+  templateUrl: './authentication.component.html',
+  styleUrls: ['./authentication.component.scss']
+})
+export class AuthenticationComponent implements OnInit {
+  public loginName: string;
+  public loginPassword: string;
+  
+  constructor(private authService: AuthService) { }
+
+  ngOnInit(): void {
```

Comment:
It's better to get rid of empty methods

Reply:
Ok. I'll make it in the next pull request